src/v0/destinations/hs/util.js

```diff
@@ -533,7 +524,7 @@ const getExistingData = async (inputs, destination) => {
       Config.authorizationType === 'newPrivateAppApi'
         ? endpoint
         : `${endpoint}?hapikey=${Config.apiKey}`;
-    searchResponse =
+    const searchResponse =
```

Comment:
move: 
const endpoint = IDENTIFY_CRM_SEARCH_ALL_OBJECTS.replace(':objectType', objectType);
    const endpointPath = `objects/:objectType/search`;
    const url =
      Config.authorizationType === 'newPrivateAppApi'
        ? endpoint
        : `${endpoint}?hapikey=${Config.apiKey}`;
        
       these outside the while block

Reply:
addressed